deepspeed/runtime/pipe/engine.py

```diff
@@ -451,8 +457,11 @@ def eval_batch(self, data_iter, return_logits=False, compute_loss=True, reduce_o
         train_iterator = self.data_iterator
         self.set_dataiterator(data_iter)
 
+        # set the number micro batches incase the user chose value than training
```

Comment:
The formatting check is failing on codespell [here](https://github.com/microsoft/DeepSpeed/actions/runs/7391638608/job/20108766055?pr=4859#step:5:71), "incase" -> "in case"

Reply:
Thanks, fixed it :)